concepticondata/conceptlists/Lai-2023-291.tsv

```diff
@@ -0,0 +1,292 @@
+Lai-2023-291-36	241	the wolf	522	WOLF	324	the wolf
+Lai-2023-291-37	3	the ant	587	ANT	365	the ant
+Lai-2023-291-38	387	to hide (conceal)	602	HIDE (CONCEAL)		to hide (conceal)
+Lai-2023-291-39	77	the grass	606	GRASS	436	the grass
+Lai-2023-291-40	79	the hail	609	HAIL	13	the hail
+Lai-2023-291-41	95	the horse	615	HORSE	268	the horse
+Lai-2023-291-42	102	the ice	617	ICE	17	the ice
+Lai-2023-291-43	109	the lake	624	LAKE	31	the lake
+Lai-2023-291-44	112	the leaf	628	LEAF	376	the leaf
+Lai-2023-291-45	126	the meat	634	MEAT	443	the meat
+Lai-2023-291-46	132	the mountain	639	MOUNTAIN	23	the mountain
+Lai-2023-291-47	135	the mud	640	MUD	46	the mud
+Lai-2023-291-48	295	the ashes	646	ASH		the ashes
+Lai-2023-291-49	156	the rain	658	RAIN (PRECIPITATION)	10	the rain
+Lai-2023-291-50	162	the river	666	RIVER	30	the river
+Lai-2023-291-51	163	the road	667	ROAD	38	the road
+Lai-2023-291-52	164	the root	670	ROOT	375	the root
+Lai-2023-291-53	170	the sand	671	SAND	44	the sand
+Lai-2023-291-54	43	early	672	EARLY	1018	early
+Lai-2023-291-55	134	the mouth	674	MOUTH	84	the mouth
+Lai-2023-291-56	250	the shit	676	SHIT (DEFECATE)	151	the shit
+Lai-2023-291-57	343	the person	683	PERSON		the person
+Lai-2023-291-58	122	the lung	688	LUNG	143	the lung
+Lai-2023-291-59	31	dark	706	DARK	1013	dark
+Lai-2023-291-60	201	to steal	713	STEAL	1686	to steal
+Lai-2023-291-61	174	the seed	714	SEED	405	the seed
+Lai-2023-291-62	192	the snake	730	SNAKE	347	the snake
+Lai-2023-291-63	47	the egg	744	EGG	450	the egg
+Lai-2023-291-64	1597	to sow (broadcast, scatter seeds)	748	SOW SEEDS	244	to sow (broadcast, scatter seeds)
+Lai-2023-291-65	150	outside	762	OUTSIDE	714	outside
+Lai-2023-291-66	185	the skin	763	SKIN	120	the skin
+Lai-2023-291-67	346	the roof	769	ROOF		the roof
+Lai-2023-291-68	190	the smoke	778	SMOKE (EXHAUST)	19	the smoke
+Lai-2023-291-69	193	the snow	784	SNOW	12	the snow
+Lai-2023-291-70	211	the thigh	471	UPPER LEG (THIGH)	100	the thigh
+Lai-2023-291-71	197	the spider	843	SPIDER	361	the spider
+Lai-2023-291-72	219	the tiger	846	TIGER	304	the tiger
+Lai-2023-291-73	203	the stone (a piece of)	857	STONE	43	the stone (a piece of)
+Lai-2023-291-74	224	the tree	906	TREE	372	the tree
+Lai-2023-291-75	7	the barley (tibetan or highland)	932	BARLEY	411	the barley (tibetan or highland)
+Lai-2023-291-76	12	the bird	937	BIRD	326	the bird
+Lai-2023-291-77	15	the blood	946	BLOOD	129	the blood
+Lai-2023-291-78	229	the water	948	WATER	47	the water
+Lai-2023-291-79	238	the wind	960	WIND	9	the wind
+Lai-2023-291-80	242	the woman	962	WOMAN	174	the woman
+Lai-2023-291-81	161	right	1019	RIGHT	711	right
+Lai-2023-291-82	425	to yawn	1028	YAWN		to yawn
+Lai-2023-291-83	76	good	1035	GOOD	1052	good
+Lai-2023-291-84	80	the hair (of the head)	1040	HAIR	75	the hair (of the head)
+Lai-2023-291-85	117	light (of weight)	1052	LIGHT (WEIGHT)	1015	light (of weight)
+Lai-2023-291-86	233	the wheat	1077	WHEAT	410	the wheat
+Lai-2023-291-87	359	the work	1086	WORK (ACTIVITY)		the work
```

Comment:
The current mapping referes to the action. [WORK (LABOUR)](https://concepticon.clld.org/parameters/984) is a more suitable mapping.